### ISSUE TITLE
widget crash after i press the button to login

### ISSUE THREAD

**Issue by Nikitarex** [joshhunt/GhostOverlay#3]
As the title says, fresh new install of the widget and it crashes when i try to log in.

**Comment by ppkhoa** [comment]
Same issue here, currently on Windows 10 1909 (18363.836)

**Comment by joshhunt** [comment]
Yup, for now, you can use this link to log in https://www.bungie.net/en/OAuth/Authorize?client_id=32459&response_type=code

Does the widget crash with "something went wrong" immediately as you click the Login with Bungie? Do you ever see a login window pop up? 

What Windows version (and build) are you running? Do you have any Windows updates pending? Any updates in the Microsoft Store pending?

**Comment by Nikitarex** [comment]
> Yup, for now, you can use this link to log in https://www.bungie.net/en/OAuth/Authorize?client_id=32459&response_type=code
> 
> Does the widget crash with "something went wrong" immediately as you click the Login with Bungie? Do you ever see a login window pop up?
> 
> What Windows version (and build) are you running? Do you have any Windows updates pending? Any updates in the Microsoft Store pending?

Actually i re-tried after few hours and it worked without the link... how strange...
However i'm on windows 10 pro 1909 build 18363.836 and no updates.
Don't know why it suddenly worked now because i haven't done anything and there weren't updates... so... i might close the issue

**Comment by joshhunt** [comment]
don't close. it's still and issue i need to fix :)

**Comment by ppkhoa** [comment]
> Yup, for now, you can use this link to log in https://www.bungie.net/en/OAuth/Authorize?client_id=32459&response_type=code
> 
> 
> 
> Does the widget crash with "something went wrong" immediately as you click the Login with Bungie? Do you ever see a login window pop up? 
> 
> 
> 
> What Windows version (and build) are you running? Do you have any Windows updates pending? Any updates in the Microsoft Store pending?

The widget did crash immediately after clicking login with Bungie and close out Game Bar with no window popped up. Opened Game Bar again and the widget showed “something went wrong” and a restart button.

Windows 10 1909 (18363.836)
